providers/screensaver.rb

```diff
@@ -31,39 +31,31 @@
   ## Distinguir entre sesion Cinnamon y LXDE
 
   #     session = node["desktop_session"] 
-        gecos_ws_mgmt_desktop_setting "idle-activation-enabled" do
-          type "string"
+        desktop_gsettings "org.cinnamon.desktop.screensaver" do
```

Comment:
Avoid reusing a resource name ("org.cinnamon.desktop.screensaver"). Append the key name to this resource.